_dashboards/index.md

```diff
@@ -30,14 +34,7 @@ OpenSearch Dashboards gives users the ability to organize their logs, traces, an
 
 # Install OpenSearch Dashboards
 
-The basic installation of Dashboards includes the following features:
-- [Discover](https://github.com/opensearch-project/documentation-website/issues/991): Lets you see the OpenSearch data.
-- [Visualize](https://github.com/opensearch-project/documentation-website/issues/992): Lets you create visualizations for the data.
-- [Dashboard](https://github.com/opensearch-project/documentation-website/issues/941): Displays multiple visualizations on a dashboard.
-
-The default installation of Dashboards includes a full suite of [OpenSearch plugin](https://opensearch.org/docs/latest/dashboards/install/plugins/) features that lets you monitor events, identify trends, and automate recurring activities.
-
-For more information, see [OpenSearch Installation Instructions](https://opensearch.org/docs/latest/opensearch/install/index/).
+After installing OpenSearch, you can install OpenSearch Dashboards, with different installation methods to choose from. For more information, see [OpenSearch Installation Instructions](https://opensearch.org/docs/latest/opensearch/install/index/).
```

Comment:
The name of the link should be `Install and configure OpenSearch` so it aligns with the header on the page. You also need to make sure that the base URL is dynamic, since you don't know which version the user is browsing. By hard-coding in a URL with `/latest/` in the path you would force all users to the newest version supported by the website.

Change the link part to:

[Install and configure OpenSearch]({{site.url}}{{site.baseurl}}/opensearch/install/index/)